### DISCUSSION TITLE
I need know how to deploy the installing enviroment.

### DISCUSSION THREAD

**Discussion by Ngsc-team** [AntonOsika/gpt-engineer#271] [Q&A]
Hi everyone,I am interested in this gpt-engineer and want to test if it ca do some sophiscated and complited programming task.

Met error preparing enviroment now,

PS,I have installed conda for google research football but I forgot it is python combined or not.

the error log is : 

PS C:\gpt-engineer> git:(main) conda create --name gpt-eng python-3,11,3
git: : The term 'git:' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
 spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ git:(main) conda create --name gpt-eng python-3,11,3
+ ~~~~
    + CategoryInfo          : ObjectNotFound: (git::String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

please help.thank you

Regards.

**Comment by delirans** [comment]
You need to install git: https://git-scm.com/downloads

**Comment by patillacode** [comment]
If you wish to reopen the issue please do following the new issue template.
